cptoml.py

```diff
@@ -156,57 +167,51 @@ def _tablefind(buf, subtable) -> int:
 
     Returns -1 when not found.
     """
-    result = -1
-    tm = 0
-    q = True
-    while q:
-        tml = None
+
+    for index, line in enumerate(buf):
         try:
-            tml = _prepareline(buf[tm])
+            tml = _prepareline(line)
+
+            if tml == f"[{subtable}]":
+                return index
+
         except IndexError:
```

Comment:
Not needed?

Reply:
Comments may be after the thingy.
`[subtable whatever] # Explaining subtable`
`.startswith(subtable)`

Reply:
Oh the IndexError itself is there for safety. During development it was needed.
Once I get testing ci going, we can try removing it.

Reply:
> Comments may be after the thingy.

On the file yeah, on this code we've already cleaned then up with `_prepareline`